plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -429,7 +429,15 @@ class ServicesTable extends React.Component {
         ? FrameworkUtil.extractBaseTechVersion(rawVersion)
         : "";
 
-    return <Tooltip content={rawVersion}>{displayVersion}</Tooltip>;
+    return (
```

Comment:
This would be something we would need to do anywhere we use tooltips within the new table? Is there maybe a more general solution? not a blocker for this PR, just a thought.

Reply:
A more general solution would be good, I just have to think about what that might look like. A more general solution would probably happen in ui-kit's Table styles